README.md

```diff
@@ -29,11 +44,84 @@ $ pip install pytest-raises
 Usage
 -----
 
-Marking a test with the `@pytest.mark.raises()` decorator will mark that the code the test executes is expected to raise an error.  This is different from `@pytest.mark.xfail()` as it does not mean the test itself might fail, but instead that the "pass" for the test is that the code raises an error.
-
-It accepts an `exception` keyword argument, which is the class of error expected to be raised.
-
-It will allow tests which raise errors to pass.  The main usage is to assert that an error of a specific type is raise.
+Marking a test with the `@pytest.mark.raises()` or
```

Comment:
I love these additions - this makes the functionality of the plugin much clearer!